config/sync/xmlsitemap.settings.node.authors.yml

```diff
@@ -0,0 +1,3 @@
+status: true
+priority: 0.5
```

Comment:
The priority should be 1 and changefreq daily, just like on the current site. This is only the default, it will change as content is updated /admin/config/search/xmlsitemap/settings/node/article

Reply:
updated to 1 and to daily